image.go

```diff
@@ -264,6 +267,27 @@ func Watermark(buf []byte, o ImageOptions) (Image, error) {
 	return Process(buf, opts)
 }
 
+func watermarkImage(buf []byte, o ImageOptions) (Image, error) {
+	response, err := http.Get(o.Image)
+	if err != nil {
+		return Image{}, NewError("Invalid watermark image.", BadRequest)
+	}
+	defer response.Body.Close()
+
+	imageBuf, _ := ioutil.ReadAll(response.Body)
```

Comment:
I'm not sure what other limits we employ within Imaginary, but at the very least we need to add a `io.LimitReader()` here.

Reply:
I think bytes limit should be also parametrized. Let's default to `1e6` = `1MB`.